docs/live-sim.mdx

```diff
@@ -0,0 +1,98 @@
+You can share your machine when in Live simulation mode by selecting the option **copy link** and then a modal will pop up to confirm the link was copied to your clipboard. Go ahead and share that link to other participants! 
+The link can also be accessed by users who are not logged in to the Studio.
+
+<p>(TODO:add screenshot here)
+<ThemedImage
+  alt="When selecting Start a live simulation, you are actively sharing the machine and others can access the same simulation with a custom generated link."
+  sources={{
+    light: useBaseUrl('/live-sim/live-sim-start.png'),
+    dark: useBaseUrl('/live-sim/live-sim-start-dm.png'),
+  }}
+/>
+</p>
+
+::: tip
+
+To recognize you're in a Live simulation mode, you'll notice the blue states have now trasnitioned to orange.
+
+:::
+
+4. When selecting the option **Join a Live simulation**, you'll be prompted to enter a code to reference which machine simulation session you'll be joining.
+If you were given a link to join the Live simulation session, you would only need to copy and paste that link into your browser.
+The two buttons that pop up are to **Cancel** or **Join**, when you have joined a Live simulation session. 
+
+<p>
+<ThemedImage
+  alt="Joining a Live simulation session will all you to select the states or watch them trasniton to other states with others in the session."
+  sources={{
+    light: useBaseUrl('/live-sim/live-sim-join.png'),
+    dark: useBaseUrl('/live-sim/live-sim-join-dm.png'),
+  }}
+/>
+</p>
+
+### Benefits
+
+Being able to have live participants and discussions while simulating the logic flow of your application is so important. How would you know if everyone is on the same page, unless there's a visual diagram to guide each process in a machine? 
+With Live Simulation mode, each user with the machine link can provide valuable feedback on how the machine is accurately representing the logic or is missing some states that can be easily added in using the Design mode of the Studio.
+
+<p>
+<ThemedImage
+  alt="Two views of the Studio, left is the user sharing the Live simualtion and right is the user who joins the Live simulation."
+  sources={{
+    light: useBaseUrl('/live-sim/live-sim-side-by-side.png'),
+    dark: useBaseUrl('/live-sim/live-sim-side-by-side.png'),
+  }}
+/>
+</p>
+
+Testing your application logic, collaboratively is very powerful.
+Each participant is able to interact with the machine, while other members with the shared link are able to see in real time, which part of the machine is being discussed in detail. With scrensharing, we risk having low quality images, small fonts, or pixelated text, and only one user has control over the machine at a time. Live Simulation mode is an easy solution for all of your team, whether they have a technical background or not. 
+Live Simulation mode is just the beginning or our collaborative features. If you have any ideas, suggestions, or feedback, please submit them to feedback.stately.ai 
+
+Happy Modeling!
```

Comment:
This last section beginning with "Live Simulation mode is just the beginning or our collaborative features" and ending here with "Happy Modeling" is great, but to me, it feels a bit out of place in the documentation. It would be great in the blog post to go with this though. What do you think?

Reply:
Agreed. We don’t really need to summarise docs in the same ways as blog posts. But we should totally keep this for the blog post!